src/kontrol/foundry.py

```diff
@@ -387,8 +387,14 @@ def get_optional_proof(self, test_id: str) -> Proof | None:
             return Proof.read_proof_data(self.proofs_dir, test_id)
         return None
 
-    def get_method(self, test: str) -> Contract.Method:
+    def get_method(self, test: str) -> Contract.Method | Contract.Constructor:
         contract_name, method_name = test.split('.')
+
+        if method_name == 'init':
+            constructor = self.contracts[contract_name].constructor
+            assert constructor is not None
```

Comment:
Should we throw an error if a constructor is not present? E.g., something like
```python
raise ValueError(Contract {contract.name} does not have a constructor.')
```

Reply:
Fixed.